app/views/questionnaire.py

```diff
@@ -280,6 +280,7 @@ def get_thank_you(schema, metadata, eq_id, form_type):  # pylint: disable=unused
                                      survey_title=TemplateRenderer.safe_content(schema.json['title']),
                                      is_view_submitted_response_enabled=is_view_submitted_response_enabled(schema.json),
                                      view_submission_url=view_submission_url,
+                                     account_service_url=metadata_context.get('account_service_url'),
```

Comment:
I thought we were going to always use session?

Reply:
oh, good spot!  Thought I'd updated them all